log/logger_types.go

```diff
@@ -90,7 +90,7 @@ type Levels struct {
 type SubLogger struct {
 	name string
 	Levels
-	output io.Writer
+	Output io.Writer
```

Comment:
Would it be possible instead to add a method to set a new io.Writer? Just so we can use the `RWM = &sync.RWMutex{}` in `logger_types.go` and make a bit more protected?

Reply:
sure, sounds good